src/MLDeployment.php

```diff
@@ -30,6 +34,11 @@ public static function fromArray(array $in): self
             ->setError($in['error'] ?? '')
             ->setCreatedTimestamp($in['createdTimestamp'] ?? '')
             ->setUpdatedTimestamp($in['updatedTimestamp'] ?? '');
+
+        if (!empty($in['trackingTokenId'])) {
```

Comment:
There is one minor inconsistency left. If you do `fromArray(['trackingTokenId' => '')` (empty string), the property is not set and `getTrackingTokenId`/`toArray` does not return it. To make it consistent, you should use `isset` instead of `empty` (the `??` operator works like `isset`)

Reply:
I had it that way, but it caused an issue because it doesn't want you to ever set the tokenId to empty string, you have to use the clear method, so I think this is more correct actually.

Reply:
ah,. maybe you're right